--- a/tests/OAuth/BearerValidatorTest.php
+++ b/tests/OAuth/BearerValidatorTest.php
@@ -38,7 +38,7 @@
     /**
      * @return void
      */
-    public function setUp()
+    public function setUp() : void
     {
         $this->dateTime = new DateTime('2018-01-01');
 
--- a/tests/StorageTest.php
+++ b/tests/StorageTest.php
@@ -23,7 +23,7 @@
     /**
      * @return void
      */
-    public function setUp()
+    public function setUp() : void
     {
         $dateTime = new DateTime('2018-01-01 13:37:00');
         $db = new PDO('sqlite::memory:');
--- a/tests/VpnPortalModuleTest.php
+++ b/tests/VpnPortalModuleTest.php
@@ -30,7 +30,7 @@
     /**
      * @return void
      */
-    public function setUp()
+    public function setUp() : void
     {
         $schemaDir = \dirname(__DIR__).'/schema';
         $serverClient = new ServerClient(new TestHttpClient(), 'serverClient');
