diff --git a/tests/OAuth/BearerValidatorTest.php b/tests/OAuth/BearerValidatorTest.php
index 1feb480..ac8526b 100644
--- a/tests/OAuth/BearerValidatorTest.php
+++ b/tests/OAuth/BearerValidatorTest.php
@@ -38,7 +38,7 @@ class BearerValidatorTest extends TestCase
     /**
      * @return void
      */
-    public function setUp()
+    public function setUp() : void
     {
         $this->dateTime = new DateTime('2018-01-01');
 
diff --git a/tests/StorageTest.php b/tests/StorageTest.php
index 8556e88..2d6ed8d 100644
--- a/tests/StorageTest.php
+++ b/tests/StorageTest.php
@@ -23,7 +23,7 @@ class StorageTest extends TestCase
     /**
      * @return void
      */
-    public function setUp()
+    public function setUp() : void
     {
         $dateTime = new DateTime('2018-01-01 13:37:00');
         $db = new PDO('sqlite::memory:');
diff --git a/tests/VpnPortalModuleTest.php b/tests/VpnPortalModuleTest.php
index 7de6254..b24d9c3 100644
--- a/tests/VpnPortalModuleTest.php
+++ b/tests/VpnPortalModuleTest.php
@@ -30,7 +30,7 @@ class VpnPortalModuleTest extends TestCase
     /**
      * @return void
      */
-    public function setUp()
+    public function setUp() : void
     {
         $schemaDir = \dirname(__DIR__).'/schema';
         $serverClient = new ServerClient(new TestHttpClient(), 'serverClient');
